src/main/config/config_eeprom.c

```diff
@@ -128,10 +107,10 @@ bool isEEPROMContentValid(void)
         return false;
     }
 
-    chk = updateChecksum(chk, header, sizeof(*header));
+    uint16_t crc = crc16_ccitt_update(0, header, sizeof(*header));
```

Comment:
It may be better to start from non-zero value(`~0` is popular choice). With zero as start, resulting CRC is independent of number of leading zeroes (but that is probably OK in eeprom case with fixed/known length)